### PR TITLE
Add icon support to glance card

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

To be more in line with `entities` card. No editor support yet, coming in a separate PR (is also still missing for the `entities` card).

**demo-hui-glance-card**
image

**demo-hui-entity-filter-card**
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/16163

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ DO NOT DELETE ANY TEXT from this template! Otherwise, your issue may be closed w
 ## Problem-relevant frontend configuration
 
 <!--
-  An example configuration that caused the problem for you, e.g. the YAML configuration 
-  of the used cards. Fill this out even if it seems unimportant to you. Please be sure 
+  An example configuration that caused the problem for you, e.g. the YAML configuration
+  of the used cards. Fill this out even if it seems unimportant to you. Please be sure
```
Comment (spacegaier) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:81]: No real idea why the merge brought that one in...

**Comment by bramkragten** [comment]
Personally, I really don't like this option, and would rather see it removed then expanded.

**Comment by spacegaier** [comment]
> 
> 
> Personally, I really don't like this option, and would rather see it removed then expanded.

"This option" = option to specify an icon on any card? I personally find it a nice option, but it should be consistent (hence this PR and also the one that shows the entity icon for a gauge). If we do not move forward with this PR, then we should probably be consistent and remove the option from other cards as well, but that might get some understandable pushback and I personally feel is a regression overall.

What don't you like about this option?

**Comment by zsarnett** [comment]
It doesn't look nice. And isnt really necessary. We should just remove it all together. In agreement with Bram

**Comment by spacegaier** [comment]
I think it looks quite nice (especially on an `entities` card) plus I don't see a big maintenance burden for something this simple and yet flexible, but since I don't have the votes I am closing this PR 🤷 . Might rescue the gallery improvements into a separate PR.

image

**Pull request closed without merging**